index.html

```diff
@@ -59,157 +64,184 @@ <h4 class="blue-title">LET'S CONNECT</h4>
         </li>
         <li>
           <a href="#">
-            <img src="img/icons/peace.png" alt="peace" class="social-icon" />
+            <img
+              src="img/icons/angellist.png"
+              alt="angellist"
+              class="social-icon"
+            />
           </a>
         </li>
       </ul>
     </section>
-    <section class="works">
+    <section class="works-page" id="portfolio">
       <section class="work-card">
         <img
```

Comment:
### Your Margin
![Capture0](https://user-images.githubusercontent.com/93039359/199653905-a454209f-bfcd-4347-9987-b0d9f5873957.PNG)

### Figma Margin
![Capture](https://user-images.githubusercontent.com/93039359/199653917-e39e46cd-5629-4e58-b5e6-49209c87601f.PNG)

- [ ] You can see that in the Figma image above there is an space between the cards and the homepage. You didn't have that enough space. Kindly increase that space to make your design look similar to the Figma template 😊